lib/matplotlib/axes/_axes.py

```diff
@@ -2705,11 +2705,11 @@ def errorbar(self, x, y, yerr=None, xerr=None,
 
         Parameters
         ----------
-        x : scalar
-        y : scalar
+        x : scalar or array-like
```

Comment:
Since this thread is based on an outdated diff, I went ahead and changed back to `scalar or array-like`. I think that is a reasonable compromise that does not sacrifice clarity, which is really the main objective here (more so than consistency in my opinion).